.huskyrc

```diff
@@ -0,0 +1,2 @@
+hooks:
+  pre-commit: "pretty-quick --staged"
```

Comment:
so will this run prettier on every commit? :+1:

Reply:
This does! I don't want prettier rewriting source files on webpack-dev-server, and this is a good "catch" to make sure it's run automatically at least once. Folks can still run `npm run fmt` manually to run it any time. Of course we need to document this ...